docs/14_0_0/components/chart.md

```diff
@@ -0,0 +1,270 @@
+                + "        \"label\": \"Device Id: 524967 Register: A+ total Wh \",\r\n"
+                + "        \"minBarLength\": 3\r\n"
+                + "      }\r\n"
+                + "    ]\r\n"
+                + "  },\r\n"
+                + "  \"options\": {\r\n"
+                + "    \"plugins\": {\r\n"
+                + "      \"legend\": {\r\n"
+                + "        \"display\": true,\r\n"
+                + "        \"fullWidth\": true,\r\n"
+                + "        \"position\": \"top\",\r\n"
+                + "        \"reverse\": false,\r\n"
+                + "        \"rtl\": false\r\n"
+                + "      },\r\n"
+                + "      \"title\": {\r\n"
+                + "        \"display\": true,\r\n"
+                + "        \"text\": \"Values from the meter\"\r\n"
+                + "      }\r\n"
+                + "    },\r\n"
+                + "    \"scales\": {\r\n"
+                + "      \"x\": {\r\n"
+                + "        \"beginAtZero\": false,\r\n"
+                + "        \"offset\": true,\r\n"
+                + "        \"reverse\": false,\r\n"
+                + "        \"stacked\": true,\r\n"
+                + "        \"ticks\": {\r\n"
+                + "          \"autoSkip\": true,\r\n"
+                + "          \"maxRotation\": 0,\r\n"
+                + "          \"minRotation\": 0,\r\n"
+                + "          \"mirror\": false,\r\n"
+                + "          \"source\": \"data\"\r\n"
+                + "        },\r\n"
+                + "        \"time\": {\r\n"
+                + "          \"displayFormats\": {\r\n"
+                + "            \"minute\": \"dd.LL T\"\r\n"
+                + "          },\r\n"
+                + "          \"round\": \"minute\",\r\n"
+                + "          \"stepSize\": \"60\",\r\n"
+                + "          \"unit\": \"minute\"\r\n"
+                + "        },\r\n"
+                + "        \"type\": \"time\"\r\n"
+                + "      },\r\n"
+                + "      \"y\": {\r\n"
+                + "        \"beginAtZero\": false,\r\n"
+                + "        \"offset\": false,\r\n"
+                + "        \"reverse\": false,\r\n"
+                + "        \"stacked\": true,\r\n"
+                + "        \"ticks\": {\r\n"
+                + "          \"autoSkip\": true,\r\n"
+                + "          \"mirror\": false\r\n"
+                + "        }\r\n"
+                + "      }\r\n"
+                + "    },\r\n"
+                + "    \"showLine\": true,\r\n"
+                + "    \"spanGaps\": false\r\n"
+                + "  }\r\n"
+                + "}";
+    }
+
+    public String getJson() {
+        return json;
+    }
+}
+```
+
+### Getting Started with XDEV Java Model
+
+You can use [XDEV Chart.js Java Model](https://github.com/xdev-software/chartjs-java-model) to render a server side model of your Chart.
```

Comment:
should we indicate the XDEV version compatible with PF?

Reply:
Let me add it to the Dependencies section of the docs where we have OpenLibre and other optional libraries.